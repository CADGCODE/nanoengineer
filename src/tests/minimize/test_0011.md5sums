stdout 8DE644CF684F35B4E3960183AAD4BDA4
stderr D41D8CD98F00B204E9800998ECF8427E
test_0011.xyz 837CA1E942AEAD95C081B1979058809A
