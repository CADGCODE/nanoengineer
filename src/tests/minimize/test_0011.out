======= test_0011.test =======
======= exitvalue =======
0
======= stderr =======
======= stdout =======
iters per frame = 10
number of frames = 100
timestep = 1.000000e-16
temp = 0.000000
dump as text
< test_0011.mmp  > test_0011.xyz

Total Ke = 9.315000e-20
final RMS gradient=0.989736 after 149 iterations
======= test_0011.trc =======
# nanoENGINEER-1.com Simulator Trace File, Version 050310
#
# Input File:test_0011.mmp
# Output File: test_0011.xyz
# Trace File: test_0011.trc
# Number of Atoms: 15
# Number of Frames: 100
# Steps per Frame: 10
# Temperature: 300.0
# 
# 0 columns:
#

# rms force, high force
1 1295.29 1717.47
2 809.27 1214.31
3 437.00 855.46
4 392.78 783.65
5 366.55 607.92
6 275.33 450.24
7 246.42 434.50
8 219.10 363.85
9 191.20 330.61
10 166.54 248.00
11 147.12 249.90
12 130.80 201.20
13 116.33 194.27
14 112.63 182.76
15 94.11 151.46
16 85.87 125.11
17 69.34 113.62
18 63.12 99.82
19 56.76 91.69
20 50.02 79.73
21 43.13 69.57
# Switching to Conjugate-Gradient
22 17.88 29.87
23 17.62 28.84
24 11.94 14.77
25 5.95 8.93
26 5.93 8.71
27 5.82 8.07
28 5.11 8.40
29 4.92 8.18
30 4.89 7.98
31 4.88 7.84
32 4.87 7.76
33 4.87 7.73
34 4.86 7.75
35 4.86 7.79
36 4.85 7.88
37 4.84 7.98
38 4.84 8.09
39 4.83 8.17
40 4.82 8.22
41 4.81 8.22
42 4.80 8.12
43 4.79 7.96
44 4.78 7.78
45 4.78 7.63
46 4.77 7.51
47 4.76 7.42
48 4.76 7.37
49 4.76 7.37
50 4.75 7.39
51 4.75 7.43
52 4.75 7.49
53 4.74 7.54
54 4.74 7.60
55 4.74 7.68
56 4.73 7.77
57 4.73 7.88
58 4.72 7.94
59 4.72 7.96
60 4.71 7.90
61 4.69 7.80
62 4.68 7.65
63 4.67 7.50
64 4.65 7.39
65 4.63 7.35
66 4.62 7.41
67 4.59 7.55
68 4.56 7.68
69 4.51 7.61
70 4.46 7.30
71 4.41 6.90
72 4.36 6.57
73 4.32 6.48
74 4.28 6.63
75 4.24 6.74
76 4.20 6.71
77 4.17 6.55
78 4.14 6.40
79 4.10 6.37
80 4.05 6.51
81 3.99 6.68
82 3.92 6.67
83 3.81 6.24
84 3.71 5.66
85 3.59 5.45
86 3.47 5.60
87 3.39 5.64
88 3.31 5.40
89 3.25 5.01
90 3.20 4.76
91 3.16 4.70
92 3.12 4.77
93 3.09 4.95
94 3.05 5.18
95 2.99 5.18
96 2.92 4.85
97 2.83 4.41
98 2.74 4.29
99 2.65 4.39
100 2.56 4.22
101 2.50 3.88
102 2.45 3.60
103 2.41 3.70
104 2.39 3.81
105 2.37 3.86
106 2.34 3.80
107 2.31 3.66
108 2.28 3.56
109 2.24 3.59
110 2.19 3.70
111 2.13 3.66
112 2.06 3.34
113 2.00 3.01
114 1.94 2.94
115 1.89 3.03
116 1.86 3.05
117 1.83 2.97
118 1.80 2.90
119 1.78 2.78
120 1.76 2.66
121 1.74 2.69
122 1.71 2.74
123 1.68 2.84
124 1.63 2.78
125 1.59 2.54
126 1.53 2.31
127 1.48 2.34
128 1.44 2.40
129 1.41 2.32
130 1.39 2.16
131 1.37 2.07
132 1.35 2.13
133 1.34 2.18
134 1.32 2.19
135 1.30 2.13
136 1.28 2.03
137 1.26 1.88
138 1.22 1.93
139 1.18 1.98
140 1.14 1.90
141 1.10 1.74
142 1.08 1.64
143 1.06 1.67
144 1.04 1.69
145 1.03 1.66
146 1.01 1.68
147 1.00 1.64
148 0.99 1.56
149 0.99 1.56
# Done: Minimization final rms: 0.989736, highForce: 1.555392
======= test_0011.xyz =======
15
RMS=0.989736
C -2.763416 -1.043896 -0.080977
X -2.556064 -0.541942 0.491165
X -3.454373 -1.391121 0.074903
C -2.866840 -0.272646 -1.430793
X -2.706347 0.488748 -1.306376
X -3.588896 -0.371904 -1.730389
C -1.661356 -2.134389 -0.234320
X -1.090851 -2.001342 0.292744
X -1.986250 -2.845716 -0.137165
C -1.813338 -0.870650 -2.410131
X -1.295055 -0.310320 -2.606110
X -2.183153 -1.158563 -3.043634
C -1.073430 -2.025651 -1.672535
X -1.199384 -2.701366 -2.056223
X -0.306246 -1.855243 -1.627159
